src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import "./App.scss";
+import Footer from "./components/footer/Footer";
+import Header from "./components/header/Header";
+import Landing from "./components/landing/Landing";
+
+function App() {
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Landing />} />
```

Comment:
lets have a constants folder and make a routes file which stores the values of the routes. access the routes constant here.